Objects/bytes_methods.c

```diff
@@ -765,3 +715,21 @@ _Py_bytes_endswith(const char *str, Py_ssize_t len, PyObject *subobj,
 {
     return _Py_bytes_tailmatch(str, len, "endswith", subobj, start, end, +1);
 }
+
+PyDoc_STRVAR_shared(_Py_isascii__doc__,
+"B.isascii() -> bool\n\
+\n\
+Return True if B is empty or all characters in B are ASCII,\n\
+False otherwise.");
+
+PyObject*
+_Py_bytes_isascii(const char *cptr, Py_ssize_t len)
+{
+    const char *p = cptr;
+    const char *end = p + len;
+    Py_ssize_t max_char = stringlib_find_max_char(cptr, end);
```

Comment:
Can you please revert this change to make the PR easier to review? You can open a separated PR for that.

Reply:
Done. https://github.com/python/cpython/pull/120497